### DISCUSSION TITLE
Follow-up from comment on #3249

### DISCUSSION THREAD

**Discussion by wjk** [microsoft/WindowsAppSDK#3555] [General]
As requested by @DrusTheAxe.

> Are you aware of [AddPackageOptions.AllowUnsigned](https://learn.microsoft.com/uwp/api/windows.management.deployment.addpackageoptions.allowunsigned?view=winrt-22621#windows-management-deployment-addpackageoptions-allowunsigned)?

No, actually, I was not. This is amazing news.

With regards to provisioning, I was operating under mistaken assumptions. Please accept my apologies. As for the package removal issue, I really have no complaint here. I was only thinking of the case where an IT department would provision a package on a managed device, and then the user would remove it, and then complain that their app is missing (compared to traditional deployment, where only the administrator can install or remove). But if that were to happen, it is then the responsibility of the IT department to fix it, not us. 😄

**Comment by DrusTheAxe** [comment]
No worries. It's a wide and sometimes complex world we live in. Glad you found a path thru the forest that works for you :-)
